server/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "engines": {
+    "node": "8.1.1",
+    "npm": "5.0.3"
+  },
```

Comment:
so specific. 😛 

Did you have a reason for requiring a specific engine? Seems to run fine on my end with:
```console
$ node -v && npm -v
v8.11.1
5.6.0
```
NodeJS 8.11.1 is the latest LTS version (Long Term Support) and per
https://nodejs.org
...it ships with npm v5.6.0. Do you use Node Version Manager or something fancy?